models/product-type/src/generator.ts

```diff
@@ -0,0 +1,23 @@
+import * as AttributeDefinition from '@commercetools-test-data/attribute-definition';
+import { fake, Generator } from '@commercetools-test-data/core';
+import { createRelatedDates } from '@commercetools-test-data/utils';
+import { TProductType } from './types';
+
+const [getOlderDate, getNewerDate] = createRelatedDates();
+
+// https://docs.commercetools.com/api/projects/productTypes#product-types
+
+const generator = Generator<TProductType>({
+  fields: {
+    id: fake((f) => f.datatype.uuid()),
+    key: fake((f) => f.lorem.slug()),
+    version: fake((f) => f.datatype.number()),
+    createdAt: fake(getOlderDate),
+    lastModifiedAt: fake(getNewerDate),
+    name: fake((f) => f.random.words(2)),
+    description: fake((f) => f.random.words()),
+    attributes: fake(() => [AttributeDefinition.random()]),
```

Comment:
Missing `createdBy` and `lastModifiedBy`

Reply:
Should `createdBy` and `lastModifiedBy` be included in the tests, or is that double-testing?

Reply:
Yes because they are different, here's an [example](https://github.com/commercetools/test-data/blob/150b09b1d38bccc0d3cddced69fdf4c94263b04b/models/discount-code/src/builder.spec.ts#L33-L40)

Reply:
🔧  Fixed with [2f4ca81](https://github.com/commercetools/test-data/pull/198/commits/2f4ca8174a0c9595ff0eb70101921a68468f89be)